migration/src/m0000475_improve_version_comparison_fns/semver_cmp.sql

```diff
@@ -0,0 +1,128 @@
+
+create or replace function semver_cmp(left_p text, right_p text)
+    returns integer
+as
+$$
+declare
+    left_parts text[];
+    right_parts text[];
+
+    left_major bigint;
+    left_minor bigint;
+    left_patch bigint;
+    left_pre text;
+    left_build text;
+
+    right_major bigint;
+    right_minor bigint;
+    right_patch bigint;
+    right_pre text;
+    right_build text;
+
+    left_numeric bool;
+    right_numeric bool;
+
+    cur integer;
+
+begin
+
+    raise notice 'semver_cmp % %', left_p, right_p;
```

Comment:
FWIW - **log_min_messages** configuration parameter to a higher level, such as WARNING or ERROR, to suppress NOTICE messages ... useful for production when we dont want to generate gigantic log files ;)

Reply:
fwiw, I can never actually find the postgres logs. raise notice shows up in the rust log if `trace` at least.

Reply:
ya in production, you might find this jams up logs unnec ... just mentioned